exporter/datadogexporter/config.go

```diff
@@ -483,5 +500,9 @@ func (c *Config) Unmarshal(configMap *confmap.Conf) error {
 	if !configMap.IsSet("traces::endpoint") {
 		c.Traces.TCPAddr.Endpoint = fmt.Sprintf("https://trace.agent.%s", c.API.Site)
 	}
+	if !configMap.IsSet("logs::endpoint") {
+		// this is v2 endpoint
```

Comment:
Does this comment add anything? Do we need it? Why do we care that it's the v2 endpoint?

Reply:
This is v2 , as defined by datadog api client .. An example is here https://github.com/DataDog/datadog-api-client-go/blob/master/examples/v2/logs/SubmitLog.go

Reply:
V1 endpoint is completely different

Reply:
removing this